### ISSUE TITLE
Drawpile Ease Of Access Feature for font size?

### ISSUE THREAD

**Issue by ChazShot** [drawpile/Drawpile#857]
The font sizes for the description on tools are extremely small and frustrating to look at. Is there an option in Preferences where we can change the size for the font?

![Capture](https://user-images.githubusercontent.com/54880477/72203991-3c1ea900-34c7-11ea-8fba-4fa4e1ca1c59.PNG)

**Comment by callaa** [comment]
That looks like a high-DPI scaling error. Out of curiosity, does the problem also happen if you switch to dark mode? (Dark mode uses a different style which might not have the same bug.)

**Comment by notfood** [comment]
Can you start Drawpile.exe with `-style fusion`?

**Comment by ChazShot** [comment]
![Capture](https://user-images.githubusercontent.com/54880477/72213310-d7a03000-3540-11ea-9712-353f9f6937ec.PNG)
Oh, just wanted to write to you all that the problem was solved when I shut the computer down and reopened Drawpile. I'm not quite sure how the problem first started but thanks for the tip offs. (Yes I have tried drawpile in night-mode: It was fine.) (I'm not sure what -style fusion suppose to mean)  

It font size use to be way smaller than what I have from my previous image. During this bug, I even fiddled with the display settings on my computer however the text font still remains the same, extremely small. Now all is fine.
